### PR TITLE
Add RecallCard

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a> 
    * **PR #1971** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971" target="_blank">Graphite</a> 
      * **PR #1972** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1972" target="_blank">Graphite</a> 
        * **PR #1973** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1973" target="_blank">Graphite</a> 
          * **PR #1974** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1974" target="_blank">Graphite</a> 
            * **PR #1979** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1979" target="_blank">Graphite</a> 
              * **PR #1985** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1985" target="_blank">Graphite</a> 
                * **PR #2004** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2004" target="_blank">Graphite</a> 
                  * **PR #2005** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2005" target="_blank">Graphite</a> 
                    * **PR #2006** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2006" target="_blank">Graphite</a> 
                      * **PR #2007** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2007" target="_blank">Graphite</a> 
                        * **PR #2008** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2008" target="_blank">Graphite</a> 
                          * **PR #2012** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2012" target="_blank">Graphite</a> 
                            * **PR #2013** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2013" target="_blank">Graphite</a> 
                              * **PR #2014** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2014" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2014?utm_source=stack-comment).

File: src/components/Create/components/RecallCard/RecallCard.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  return (
+    <div
+      style={{
+        width: '100%',
+        padding: '0.625rem 1rem',
+        backgroundColor: colors.background.l2,
+      }}
+    >
+      {show.includes('fundingCycles') && <FundingCycleRecallOption />}
```
Comment (tomquirk) [inline src/components/Create/components/RecallCard/RecallCard.tsx:59]: extremely dank. luv

File: src/components/Create/components/RecallCard/hooks/FundingCycle.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    unit: derivedUnit,
+    plural: formattedDuration > 1,
+  })
+  return `${secondsToOtherUnit({
+    duration: durationAsNumber,
+    unit: derivedUnit,
+  })} ${formattedUnit}`
+}
+
+export const useFundingCycle = () => {
```
Comment (tomquirk) [inline src/components/Create/components/RecallCard/hooks/FundingCycle.ts:34]: **suggestion** wdyt about something like `useFundingCycleRecallValue` or similar

**Review by tomquirk — APPROVED** [review]
just one naming suggestion!

File: src/components/Create/components/RecallCard/hooks/FundingCycle.ts (hunk 2 above)
Comment (wraeth-eth) [reply src/components/Create/components/RecallCard/hooks/FundingCycle.ts:34]: love it - will make the change

**Pull request merged**